plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -969,6 +1023,8 @@ const getInteractivityAPIConfig = ( options = {} ) => {
 									'@babel/plugin-proposal-optional-chaining',
 									'@babel/plugin-proposal-class-properties',
 								],
+								cacheDirectory: true,
+								cacheCompression: false,
```

Comment:
TODO: commands to cleanup babel cache (connected to watch/build) - blocks/admin plugins.